src/sql/workbench/contrib/notebook/test/browser/notebookEditor.test.ts

```diff
@@ -56,14 +56,17 @@ import { UntitledTextEditorInput } from 'vs/workbench/services/untitled/common/u
 import { IUntitledTextEditorService } from 'vs/workbench/services/untitled/common/untitledTextEditorService';
 import { workbenchInstantiationService } from 'vs/workbench/test/browser/workbenchTestServices';
 import { IProductService } from 'vs/platform/product/common/productService';
+import { IHostColorSchemeService } from 'vs/workbench/services/themes/common/hostColorSchemeService';
+import { ColorScheme } from 'vs/platform/theme/common/theme';
+import { CellModel } from 'sql/workbench/services/notebook/browser/models/cell';
 
 class NotebookModelStub extends stubs.NotebookModelStub {
 	public contentChangedEmitter = new Emitter<NotebookContentChange>();
 	private _kernelChangedEmitter = new Emitter<nb.IKernelChangedArgs>();
 	private _onActiveCellChanged = new Emitter<ICellModel>();
 
 	get cells(): ICellModel[] {
-		return this.cells;
+		return super.cells;
```

Comment:
Should be able to just remove this function completely so it'll just call directly to the superclass.

Reply:
good call.